src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -123,15 +123,23 @@ export default class RightPanel extends React.PureComponent<Props, State> {
     this.setState({ rarity, isDirty: this.isDirty({ rarity }) })
   }
 
-  handleChangeCategory = (category: WearableCategory) => {
-    let data: WearableData = {
-      ...(this.state.data as WearableData)!,
-      category
-    }
-    // when changing the category to SKIN we hide everything else
-    if (category === WearableCategory.SKIN) {
-      data = this.setReplaces(data, [])
-      data = this.setHides(data, [])
+  handleChangeCategory = (category: WearableCategory | EmoteCategory) => {
```

Comment:
This change is unrelated to this issue but allows updating the emote category in the item editor.